calico-enterprise_versioned_docs/version-3.21-1/getting-started/install-on-clusters/windows-calico/rancher.mdx

```diff
@@ -27,10 +27,9 @@ Install $[prodnameWindows] on Rancher Kubernetes Engine (RKE).
 
 The following steps will outline the installation of $[prodname] networking on the RKE cluster, then the installation of $[prodnameWindows] on the Windows nodes.
 
-1. Install the Tigera operator and custom resource definitions.
+1. Install the Tigera $[prodname] operator and custom resource definitions.
```

Comment:
nit: this is just called 'Tigera operator' (no prodname) elsewhere

Reply:
Yes indeed. Changed globally.